src/main/java/seedu/address/logic/commands/UntagEventCommand.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PERSONS;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_EVENTS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.UidList;
+import seedu.address.model.person.Person;
+
+/**
+ * Untags customers in Person List to an event.
+ */
+public class UntagEventCommand extends Command {
+    public static final String COMMAND_WORD = "untagEvent";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Untags 1 or more persons to an event in the address book "
+            + "by the index number used in the displayed person and event lists. "
+            + "Parameters: "
+            + "EVENT INDEX (must be a positive integer) "
+            + PREFIX_PERSONS + "PERSON INDEX 1 (must be a positive integer) "
+            + "[PERSON INDEX 2 (must be a positive integer) "
+            + "PERSON INDEX 3 (must be a positive integer) ...]\n"
```

Comment:
Same as above, I think you're missing underscores for EVENT_INDEX and PERSON_INDEX_1. Also I think you can omit "must be a positive integer" for PERSON_INDEX_2 and 3 to be less repetitive.

```suggestion
            + "EVENT_INDEX (must be a positive integer) "
            + PREFIX_PERSONS + "PERSON_INDEX_1 (must be a positive integer) "
            + "[PERSON_INDEX_2 PERSON_INDEX_3 ...]\n"
```